app/web/features/search/FilterDialog.tsx

```diff
@@ -84,33 +83,33 @@ interface FilterModalFormData
 
 interface FilterDialogProps {
   isOpen: boolean;
-  onClose(): void;
   queryName: string;
-  setQueryName: Dispatch<SetStateAction<string>>;
-  setLocationResult: Dispatch<SetStateAction<GeocodeResult>>;
   lastActiveFilter: number;
-  setLastActiveFilter: Dispatch<SetStateAction<number>>;
   hostingStatusFilter: number;
-  setHostingStatusFilter: Dispatch<SetStateAction<number>>;
   completeProfileFilter: boolean;
-  setCompleteProfileFilter: Dispatch<SetStateAction<boolean>>;
   numberOfGuestFilter: number;
-  setNumberOfGuestFilter: Dispatch<SetStateAction<number>>;
+  onClose(): void;
+  setQueryName: (queryName: string) => void;
+  setLocationResult: (locationResult: GeocodeResult) => void;
+  setLastActiveFilter: (lastActive: number) => void;
+  setHostingStatusFilter: (hostingStatus: number) => void;
+  setCompleteProfileFilter: (completeProfileFilter: boolean) => void;
+  setNumberOfGuestFilter: (numberOfGuests: number) => void;
```

Comment:
Is there a need to rename these types? `Dispatch<SetStateAction<type>>` is almost the same as this, except it allows for an updater callback to be passed in too (i.e. `(previousState) => newState` instead of `newState` directly